### PR TITLE
COLOS-9: Deploy Senatus DEV version

### PR DESCRIPTION
**Pull request by iboshkov**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - "main"
+      - "development"
+      - COLOS-9-deploy
```
Comment (vergjor) [inline .github/workflows/build.yml]: please remove this branch from the list before merging

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      -
+        name: Checkout
```
Comment (vergjor) [inline .github/workflows/build.yml]: move this to the line above

File: docker-compose.yml
Comment (vergjor) [inline docker-compose.yml:1]: shouldn't this yaml file be moved to the senatus directory?

File: .github/workflows/build.yml (hunk 2 above)
Comment (gjorgji-d-cc) [reply .github/workflows/build.yml]: voopshto raboti kako sho e sega? :DD D

File: .github/workflows/build.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        uses: actions/checkout@v4
+
+      - name: Log in to registry
```
Comment (dimeskigj) [inline .github/workflows/build.yml]: Брат имаш готова акција за ова 

```yml
    - name: Login to ghcr.io
      uses: docker/login-action@v3
      with:
        registry: ghcr.io
        username: ${{ github.actor }}
        password: ${{ secrets.GITHUB_TOKEN }}
 ```

File: .github/workflows/build.yml (hunk 2 above)
Comment (iboshkov) [reply .github/workflows/build.yml]: Works, but fixed.

File: .github/workflows/build.yml (hunk 3 above)
Comment (iboshkov) [reply .github/workflows/build.yml]: Why waste time write a lot word when few word do trick?

File: .github/workflows/build.yml (hunk 1 above)
Comment (iboshkov) [reply .github/workflows/build.yml]: Removed

File: docker-compose.yml
Comment (iboshkov) [reply docker-compose.yml:1]: No, this spins up all services, not just `senatus`.

**Review by vergjor — APPROVED** [review]

**Review by gjorgi-krenkov — APPROVED** [review]

**Pull request merged**